lib/api/controllers/baseController.js

```diff
@@ -72,7 +76,13 @@ class NativeController extends BaseController {
    *
    * @returns {Promise}
    */
-  init () {
+  async init () {
```

Comment:
You should not put that in the BaseController because then every controller will make those 2 calls to `kuzzle.ask`